backend/src/main/kotlin/rocks/didit/sefilm/database/entities/User.kt

```diff
@@ -17,5 +18,6 @@ data class User(
   val email: String = "",
   val bioklubbnummer: Bioklubbnummer? = null,
   val phone: PhoneNumber? = null,
-  val avatar: String? = null
+  val avatar: String? = null,
+  val foretagsbiljetter: List<Foretagsbiljett>?
```

Comment:
Again, provide a default value and never use a nullable type on a `List`. Use `listOf()` to make an empty list instead.